[2024-04-18T23:45:22.453+0000] {processor.py:161} INFO - Started process (PID=1102) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:45:22.454+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:45:22.473+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:45:22.470+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:45:22.501+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:45:22.496+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:45:22.502+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:45:22.547+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.105 seconds
[2024-04-18T23:45:53.025+0000] {processor.py:161} INFO - Started process (PID=1119) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:45:53.028+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:45:53.041+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:45:53.037+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:45:53.058+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:45:53.056+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:45:53.058+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:45:53.091+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.076 seconds
[2024-04-18T23:46:23.667+0000] {processor.py:161} INFO - Started process (PID=1144) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:46:23.673+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:46:23.675+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:46:23.675+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:46:23.686+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:46:23.685+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:46:23.687+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:46:23.707+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.061 seconds
[2024-04-18T23:46:54.128+0000] {processor.py:161} INFO - Started process (PID=1169) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:46:54.131+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:46:54.135+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:46:54.134+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:46:54.154+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:46:54.152+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:46:54.155+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:46:54.194+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.074 seconds
[2024-04-18T23:47:24.596+0000] {processor.py:161} INFO - Started process (PID=1186) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:47:24.598+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:47:24.601+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:47:24.601+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:47:24.612+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:47:24.611+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:47:24.613+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:47:24.634+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.057 seconds
[2024-04-18T23:47:55.364+0000] {processor.py:161} INFO - Started process (PID=1211) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:47:55.367+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:47:55.380+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:47:55.378+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:47:55.405+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:47:55.401+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:47:55.406+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:47:55.480+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.130 seconds
[2024-04-18T23:48:25.852+0000] {processor.py:161} INFO - Started process (PID=1235) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:48:25.855+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:48:25.858+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:48:25.857+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:48:25.869+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:48:25.867+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:48:25.869+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:48:25.896+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.051 seconds
[2024-04-18T23:48:56.407+0000] {processor.py:161} INFO - Started process (PID=1252) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:48:56.414+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:48:56.429+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:48:56.427+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:48:56.477+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:48:56.468+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:48:56.479+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:48:56.547+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.154 seconds
[2024-04-18T23:49:27.049+0000] {processor.py:161} INFO - Started process (PID=1276) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:49:27.053+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:49:27.072+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:49:27.071+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:49:27.093+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:49:27.090+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:49:27.093+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:49:27.140+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.103 seconds
[2024-04-18T23:49:57.519+0000] {processor.py:161} INFO - Started process (PID=1300) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:49:57.524+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:49:57.546+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:49:57.544+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:49:57.570+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:49:57.568+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:49:57.571+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:49:57.610+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.097 seconds
[2024-04-18T23:50:28.010+0000] {processor.py:161} INFO - Started process (PID=1317) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:50:28.012+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:50:28.017+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:50:28.017+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:50:28.028+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:50:28.026+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:50:28.028+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:50:28.055+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.050 seconds
[2024-04-18T23:50:58.550+0000] {processor.py:161} INFO - Started process (PID=1342) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:50:58.552+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:50:58.554+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:50:58.554+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:50:58.562+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:50:58.561+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:50:58.562+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:50:58.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.033 seconds
[2024-04-18T23:51:28.929+0000] {processor.py:161} INFO - Started process (PID=1358) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:51:28.934+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:51:28.939+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:51:28.939+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:51:28.948+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:51:28.947+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:51:28.949+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:51:28.989+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.065 seconds
[2024-04-18T23:51:59.334+0000] {processor.py:161} INFO - Started process (PID=1383) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:51:59.336+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:51:59.341+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:51:59.341+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:51:59.357+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:51:59.355+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:51:59.357+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:51:59.415+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.091 seconds
[2024-04-18T23:52:29.947+0000] {processor.py:161} INFO - Started process (PID=1408) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:52:29.949+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:52:29.954+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:52:29.953+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:52:29.963+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:52:29.962+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:52:29.963+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:52:29.994+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.053 seconds
[2024-04-18T23:53:00.499+0000] {processor.py:161} INFO - Started process (PID=1425) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:53:00.504+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:53:00.508+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:53:00.508+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:53:00.520+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:53:00.518+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:53:00.520+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:53:00.565+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.073 seconds
[2024-04-18T23:53:30.895+0000] {processor.py:161} INFO - Started process (PID=1450) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:53:30.897+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:53:30.900+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:53:30.900+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:53:30.919+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:53:30.917+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:53:30.919+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:53:30.941+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.050 seconds
[2024-04-18T23:54:01.287+0000] {processor.py:161} INFO - Started process (PID=1475) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:54:01.296+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:54:01.318+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:54:01.315+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:54:01.348+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:54:01.342+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:54:01.349+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:54:01.382+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.100 seconds
[2024-04-18T23:54:31.802+0000] {processor.py:161} INFO - Started process (PID=1491) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:54:31.807+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:54:31.811+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:54:31.810+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:54:31.825+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:54:31.824+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:54:31.826+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:54:31.857+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.082 seconds
[2024-04-18T23:55:02.321+0000] {processor.py:161} INFO - Started process (PID=1516) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:55:02.330+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:55:02.335+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:55:02.334+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:55:02.345+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:55:02.344+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:55:02.346+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:55:02.363+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.053 seconds
[2024-04-18T23:55:32.816+0000] {processor.py:161} INFO - Started process (PID=1540) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:55:32.825+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:55:32.828+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:55:32.828+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:55:32.837+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:55:32.836+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:55:32.838+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:55:32.866+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.060 seconds
[2024-04-18T23:56:03.264+0000] {processor.py:161} INFO - Started process (PID=1558) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:56:03.267+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:56:03.271+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:56:03.270+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:56:03.282+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:56:03.280+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:56:03.282+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:56:03.320+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.061 seconds
[2024-04-18T23:56:33.775+0000] {processor.py:161} INFO - Started process (PID=1583) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:56:33.780+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:56:33.784+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:56:33.783+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:56:33.804+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:56:33.799+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:56:33.805+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:56:33.845+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.102 seconds
[2024-04-18T23:57:04.252+0000] {processor.py:161} INFO - Started process (PID=1600) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:57:04.257+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:57:04.266+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:57:04.263+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:57:04.277+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:57:04.275+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:57:04.278+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:57:04.306+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.061 seconds
[2024-04-18T23:57:34.728+0000] {processor.py:161} INFO - Started process (PID=1625) to work on /opt/airflow/dags/src/airflow.py
[2024-04-18T23:57:34.731+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/src/airflow.py for tasks to queue
[2024-04-18T23:57:34.741+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:57:34.739+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:57:34.755+0000] {logging_mixin.py:188} INFO - [2024-04-18T23:57:34.753+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/src/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/src/airflow.py", line 4, in <module>
    from src.create import create_newfile
  File "/opt/airflow/dags/src/create.py", line 2, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'
[2024-04-18T23:57:34.755+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/src/airflow.py
[2024-04-18T23:57:34.797+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/src/airflow.py took 0.075 seconds
